spot-client/src/common/backend/utils.js

```diff
@@ -113,7 +151,10 @@ export function fetchJoinCode(serviceEndpointUrl) {
                 throw new Error(`No 'joinCode' in the response: ${JSON.stringify(json)}`);
             }
 
-            return json.joinCode;
+            return {
+                joinCode: json.joinCode,
+                jwt: json.jwt
+            };
```

Comment:
Is it known how updates to the join code will be pulled/pushed to Spot-TV?

Reply:
not yet